BUILD_COMMANDLINE.md

```diff
@@ -52,7 +52,11 @@
      Apache Ant™ version 1.10.13 compiled on September 27 2020
      ```
 
-     We recommend using the latest stable release of Apache Ant™. If your system has an older version of Apache Ant™ installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant™ Installation Instructions](https://ant.apache.org/manual/install.html).
+     We recommend using the latest stable release of Apache Ant™. If your system has an older version of Apache Ant™ installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant™ Installation Instructions](https://ant.apache.org/manual/install.html). **MacOS users are advised to install ant using brew.**
```

Comment:
If we add the others though, this should be a bit more verbose:

```suggestion
We recommend using the latest stable release of Apache Ant™. If your system has an older version of Apache Ant™ installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant™ Installation Instructions](https://ant.apache.org/manual/install.html) or any other applicable installation instruction.

 * **macOS or Linux:** e.g. the Homebrew Package Manager offers an easy installation methods for both a JDK and Apache Ant™ .
 * **Windows:** 
 ```

Reply:
Same thing here: I cannot directly commit this suggestion, @bwbohl, because the file was outdated based on some new commits. But I just added these suggestions in https://github.com/music-encoding/music-encoding/pull/1193/commits/908788912915c3c03deea4bfeca65c9ebabcb7bf.